orm/ormer_session.go

```diff
@@ -26,11 +26,13 @@ import (
 
 func GetSession(owner string, offset, limit int, field, value, sortField, sortOrder string) *xorm.Session {
 	session := AppOrmer.Engine.Prepare()
+
 	if offset != -1 && limit != -1 {
 		session.Limit(limit, offset)
 	}
+
 	if owner != "" {
-		session = session.And("owner=?", owner)
+		session = session.And("LOWER(owner) = LOWER(?)", owner)
```

Comment:
This is gonna interfere with index usage. We either need to rebuild indices to work on lower(value). Or maybe just use owner_id in these tables.

Reply:
Can you get a plan for this query to prove your point? 
I believe previous version of this code will generate index scan and a new version will make a seq scan.